### PR TITLE
[IMP] use higher python version, disable system site packages

### PR DESCRIPTION
**Pull request by hbrunn**
see discussion in https://github.com/OCA/web/pull/909#issuecomment-380045633 and https://github.com/OCA/web/pull/889#issuecomment-379801321

### CODE REVIEW

File: sample_files/.travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ cache:
     - $HOME/.cache/pip
 
 python:
-  - "2.7"
+  - "2.7.13"
```
Comment (mreficent) [inline sample_files/.travis.yml]: But why 2.7.13 and not 2.7.14? 🤔

Comment (hbrunn) [reply sample_files/.travis.yml]: because that's what both travis' and runbot's docker images support currently

**Comment by moylop260** [comment]
I will create a new PR to add minimal postgresql version...
Regards!

**Comment by hbrunn** [comment]
oh, sorry for not following up

**Comment by moylop260** [comment]
FYI I have created the higher PostgreSQL version here:
 - https://github.com/OCA/maintainer-quality-tools/pull/548

**Pull request merged**